emnify/README.md

```diff
@@ -0,0 +1,31 @@
+## Overview
+[EMnify][1] cellular IoT connectivity platform keeps your devices
+connected and secure wherever you deploy. Our network-agnostic SIMs
+let your applications connect to more than 540 networks in over 180 countries.
+We also offer the largest LTE-M coverage on the market.
+
+Use the Datadog-EMnify integration to collect metrics and data usage from IoT EMnify devices.
+
+## Setup
+Configure streaming of Usage Data following the [integration guide][2],
+sections "Integration Steps" and "Verifying the integration".
+
+### Monitors
+
+Everybody has different usage patterns so in order for monitors to reflect your specific case you must
+define workload-based boundaries and sensitivity.
+For more information check [Forecast monitor docs][4] and [Anomaly monitor docs][5].
+
+### Dashboard
+
+You can pick a period to filter data displayed.
```

Comment:
Is there a link to the dashboard that could be provided here? Such as [this example](https://github.com/DataDog/integrations-extras/blob/4c8f758c7e9afd4680043600b278be2d5a325f93/datazoom/README.md#dashboard).

Reply:
Great advice thanks!